install/db/language_text.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE `language_text` (
   `variable` varchar(30) NOT NULL default '',
   `term` varchar(50) NOT NULL default '',
   `text` blob NOT NULL,
-  `revised_date` datetime NOT NULL default '0000-00-00 00:00:00',
```

Comment:
All other locations that insert '0000-00-00 00:00:00' should be dealt with as well, which is a part of mysql upgrade.